data/json/monsters.json

```diff
@@ -7,7 +7,8 @@
     "default_faction": "",
     "species": [ "ZOMBIE" ],
     "diff": 100,
-    "size": "MEDIUM",
+    "volume": "62500 ml",
+    "weight": "81500",
```

Comment:
Why not `"weight": 81500` instead of `"weight": "81500"`?

Reply:
And in this I will regex quote marks into oblivion ASAP.